resources/lib/ipwww_radio.py

```diff
@@ -178,7 +178,7 @@ def AddAvailableLiveStreamItem(name, channelname, iconimage):
             title = name + ' - [I][COLOR %s]%s Kbps[/COLOR] [COLOR white]%s[/COLOR] [COLOR grey]%s[/COLOR][/I]' % (
                 quality_colours[quality], quality_bitrates[quality] , location_names[location], provider_name)
 
-            PlayStream(title, url, iconimage, '', '')
+            PlayStream(name, url, iconimage, '', '')
```

Comment:
I don't think you got my point.

I don't mind if you keep the kbps for radio. It makes sense there. For video, there was quite a discussion something like a year ago if kbps or Mbps should be used for video. The result was Mbps and I don't want to change that to kbps. Thus, my request was simply to change kbps in **video** back to Mbps. So no need to change anything here.

Also, my question was what **encoding** in video stood for as this was new information in the title.